### PR TITLE
[CELEBORN-1487][Phase2] CongestionController support dynamic config

### PR DESCRIPTION
**Pull request by leixm**
### What changes were proposed in this pull request?
CongestionController support dynamic config

### Why are the changes needed?
Currently, Celeborn only supports quota management based on disk file bytes/count, and this quota management cannot cope with sudden increases in traffic, which will cause corrupt to the cluster.

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
UT.

### CODE REVIEW

**Comment by leixm** [comment]
Can you help review? @RexXiong

File: worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/CongestionController.java [hunk 1]
```diff
@@ -332,4 +340,17 @@ public UserCongestionControlContext getUserCongestionContext(UserIdentifier user
   public BufferStatusHub getConsumedBufferStatusHub() {
     return consumedBufferStatusHub;
   }
+
+  private void updateQuota(ConfigService configService) {
```
Comment (RexXiong) [inline worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/CongestionController.java:344]: nit: configService is already a member variable

Comment (leixm) [reply worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/CongestionController.java]: Done.

**Review by RexXiong — APPROVED** [review]
LGTM

**Pull request closed without merging**